### PR TITLE
Azure Deployment: skip_cleanup

### LINKED ISSUE #360: Having difficulty with Azure deployment
[Here is](https://github.com/Enttoi/enttoi-ui-web/blob/master/.travis.yml) my yml file. I spent quit a lot time on it - it seems that the parameter skip_cleanup in deploy section is ignored, as result all artifacts produced during build are not deployed. 

I tried deploying via script by pushing to remote and it is working (/dist presented in source and destination).

### PR DESCRIPTION
**Pull request by felixrieseberg**
Closes #360

### CODE REVIEW

File: lib/dpl/provider/azure_webapps.rb [hunk 1]
```diff
@@ -29,6 +29,9 @@ def check_auth
       def push_app
         log "Deploying to Azure Web App '#{config['site']}'"
 
+        if options[:skip_cleanup]
+          context.shell "git add --all --force > /dev/null 2>&1"
+
```
Comment (BanzaiMan) [inline lib/dpl/provider/azure_webapps.rb]: I don't think this is valid syntax.

**Comment by felixrieseberg** [comment]
Updated.

File: lib/dpl/provider/azure_webapps.rb [hunk 2]
```diff
@@ -29,6 +29,10 @@ def check_auth
       def push_app
         log "Deploying to Azure Web App '#{config['site']}'"
 
+        if options[:skip_cleanup]
+          context.shell "git add --all --force > /dev/null 2>&1"
+          context.shell "git commit -m \"Skip Cleanup Commit\" > /dev/null 2>&1"
+
```
Comment (BanzaiMan) [inline lib/dpl/provider/azure_webapps.rb]: Hmmm. It's still wrong. There should be `end` here.

**Comment by felixrieseberg** [comment]
Ugh, that's what I get for trying to quickly do this from a phone. It's now updated and actually tested (https://travis-ci.org/felixrieseberg/dpl-azure-test).

**Comment by jenyayel** [comment]
Works as expected now. Thank you @BanzaiMan and @felixrieseberg.

**Pull request merged**